src/helpers/Environment.ts

```diff
@@ -75,19 +78,19 @@ export class EnvHelper {
   }
 
   /**
-   * @returns {Array} Array of websocket addresses or empty set
+   * @returns {Array} Array of node url addresses or empty set
+   * node url addresses can be whitespace-separated string of "https" addresses
+   * - functionality for Websocket addresses has been deprecated due to issues with WalletConnect
+   *     - WalletConnect Issue: https://github.com/WalletConnect/walletconnect-monorepo/issues/193
    */
-  static getSelfHostedSockets() {
-    let WS_LIST: string[];
-    if (
-      EnvHelper.env.REACT_APP_SELF_HOSTED_WEBSOCKETS &&
-      EnvHelper.isNotEmpty(EnvHelper.env.REACT_APP_SELF_HOSTED_WEBSOCKETS)
-    ) {
-      WS_LIST = EnvHelper.env.REACT_APP_SELF_HOSTED_WEBSOCKETS.split(new RegExp(EnvHelper.whitespaceRegex));
+  static getSelfHostedNode() {
+    let URI_LIST: string[];
+    if (EnvHelper.env.REACT_APP_SELF_HOSTED_NODE && EnvHelper.isNotEmpty(EnvHelper.env.REACT_APP_SELF_HOSTED_NODE)) {
```

Comment:
**IMPORTANT** rename REACT_APP_SELF_HOSTED_WEBSOCKETS to REACT_APP_SELF_HOSTED_NODE to reflect that we no longer are utilizing websockets.